utils/build/build_tracer_buddies.sh

```diff
@@ -9,9 +9,15 @@ docker buildx build --load --progress=plain -f utils/build/docker/java/spring-bo
 docker buildx build --load --progress=plain -f utils/build/docker/ruby/rails70.Dockerfile -t datadog/system-tests:ruby_buddy-v0 .
 docker buildx build --load --progress=plain -f utils/build/docker/golang/net-http.Dockerfile -t datadog/system-tests:golang_buddy-v0 .
 
-# Leave this line commented, it's only used when we need to push a new version of the buddy
-# docker push datadog/system-tests:python_buddy-v0
-# docker push datadog/system-tests:nodejs_buddy-v0
-# docker push datadog/system-tests:java_buddy-v0
-# docker push datadog/system-tests:ruby_buddy-v0
-# docker push datadog/system-tests:golang_buddy-v0
+
+if [ -z "$PUSH_IMAGES" ]
```

Comment:
Just for CLI quality, I think it's better to use argument, rahter than env var : 

```suggestion
if [ "$1" = "--push" ]; then
```

And the use in the calling script : 

```bash
./utils/build/build_tracer_buddies.sh --push
```